src/main/resources/assets/paths/cancelCollectionByCollectionId.yaml

```diff
@@ -0,0 +1,25 @@
+put:
```

Comment:
The ticket calls for a DELETE, but I think PUT is more accurate here, thanks for the catch. Can we also update the ticket so we can track this decision better?